### PR TITLE
[TMP] update asset uploader example to make it actually working

### PR DESCRIPTION
**Pull request by jillxuu**
### Description
add detailed examples on how to use this asset uploader.

### Test Plan
verified working
```
Starting fund node
fund response {
  reward: '{"gasUnitPrice":100,"maxGasAmount":12}',
  target: '0xfa3e389d045e057145e29f544f9809c28399f8167e0e45fd157084858dda8a0f',
  quantity: '1000000',
  id: '0x61ae083c05203ab7393eba35e2bca719c5768569e05ab0f87484c7bef400088e'
}
Collection Media URL: https://arweave.net/0C8IiWUA_hQUXNIuPCsV1Ndu2reW7J5QkzLfHH4zsz8
Collection metadata json URL: https://arweave.net/YqYjZAkd91PEhy6CiYdfC_swCNTJ5K5curMGW5m2lrg
Token media URL folder: https://arweave.net/2NhlIkaemZNc5G5gcumMbmYkCvRoQaeFTUyylZXXfOw
Token metadata json URL folder: https://arweave.net/n_8e-WepY_nfrBi2lkDu5fPyPRumucVm7HDeAkkLiT4
```

### CODE REVIEW

File: src/plugins/assetUploader/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+## How to Use
+
+Below is a sample TypeScript script that demonstrates how to use the uploader:
+
+```ts
+const example = async () => {
+    const account = Account.generate();
+    const aptosConfig = new AptosConfig({
+        network: Network.DEVNET,
```
Comment (0xmaayan) [inline src/plugins/assetUploader/README.md:48]: wait, does that work? Irys does not support Aptos devnet, need to use tesnet. SDK should throw if you configured the SDK to use testnet and trying to use Irys
take a look at https://github.com/aptos-labs/aptos-ts-sdk/blob/tmp/examples/typescript/asset_uploader.ts

Comment (jillxuu) [reply src/plugins/assetUploader/README.md]: you're right, updated to testnet

File: src/plugins/assetUploader/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+1. SDK version:
+   Ensure you have version 1.9.1-tmp.0 of `@aptos-labs/ts-sdk`.
+   [SDK Package](https://www.npmjs.com/package/@aptos-labs/ts-sdk/v/1.9.1-tmp.0?activeTab=versions)
+
+2. Install the SDK with `pnpm`:
+   ```bash
+   pnpm install @aptos-labs/ts-sdk@1.9.1-tmp.0
+   ```
+
+3. Install the Irys SDK with `npm`:
```
Comment (0xmaayan) [inline src/plugins/assetUploader/README.md:16]:
```suggestion
3. Install the Irys SDK with `pnpm`:
```

File: src/plugins/assetUploader/README.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+   ```bash
+   npm install @irys/sdk
```
Comment (0xmaayan) [inline src/plugins/assetUploader/README.md:18]:
```suggestion
   pnpm install @irys/sdk
```

File: src/plugins/assetUploader/README.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
+# NFT Asset Uploader Guide
+
+## Pre-requisite
+
+Before you begin, make sure you have the following:
+
+1. SDK version:
```
Comment (0xmaayan) [inline src/plugins/assetUploader/README.md:7]: this should not be a step, installing the package (step 2) creates it

File: src/plugins/assetUploader/README.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+Below is a sample TypeScript script that demonstrates how to use the uploader:
+
+```ts
+/* eslint-disable no-console */
+import { Account, AptosConfig, Network, Aptos, AssetUploader } from "@aptos-labs/ts-sdk";
+import * as fs from "fs";
+import * as path from "path";
+
+const example = async () => {
```
Comment (0xmaayan) [inline src/plugins/assetUploader/README.md:50]: feels a bit redundant to put the same example here, I would keep the example file only and link to it from here

File: examples/typescript/asset_uploader.ts [hunk 6]
```diff
@@ -13,27 +41,56 @@ const example = async () => {
...
+
+  // Upload the updated token metadata json folder
+  const uploadMetadataFolderResponse = await assetUploader.uploadFolder({
+    account,
+    folder: tokenMetadataJsonFolderPath,
   });
-  console.log("uploadFolderResponse", uploadFolderResponse);
-  process.exit(0);
+  const tokenMetadataJsonManifestID = uploadMetadataFolderResponse!.id;
+  const tokenMetadataJsonFolderURI = `https://arweave.net/${tokenMetadataJsonManifestID}`;
```
Comment (0xmaayan) [inline examples/typescript/asset_uploader.ts:90]: ideally, that should come from Irys response (I brought it up with the Irys team) but until then maybe we can add it to the response from irys.uploadFolder (dont need to be part of this PR)

File: src/plugins/assetUploader/README.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+    // DONE
+};
+
+example();
+```
+
+To use the provided example code, follow these steps:
+1. Replace the placeholders with the correct paths to your files.
+2. Run the script in a TypeScript-supported environment.
```
Comment (0xmaayan) [inline src/plugins/assetUploader/README.md:146]:
```suggestion
2. Run the script in a TypeScript-supported Node environment.
```

**Review by 0xmaayan — APPROVED** [review]

**Pull request merged**